src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -44,10 +45,10 @@
      *            The id of this Backpack
      */
     public PlayerBackpack(@Nonnull PlayerProfile profile, int id) {
-        this(profile, id, profile.getConfig().getInt("backpacks." + id + ".size"));
+        this(profile, id, profile.getConfig().getInt( CONFIG_PREFIX+ id + ".size"));
```

Comment:
Please place that space after the variable.
```suggestion
        this(profile, id, profile.getConfig().getInt(CONFIG_PREFIX + id + ".size"));
```

Reply:
64c639ef8